data/supertuxkart.appdata.xml

```diff
@@ -16,7 +16,7 @@
     <p>
       Discover the mystery of an underwater world, or drive through the jungles 
       of Val Verde and visit the famous Cocoa Temple. Race underground or in a
-      spaceship, under the snow of the mountain or the sun of the desert.
+      spaceship, through a rural farmland or an strange alien planet.
```

Comment:
pretty sure that should be "a strange alien planet"

Reply:
Fixed.